src/general/StringUtils.cs

```diff
@@ -187,4 +187,38 @@ public static string Unescape(this string input)
 
         return result.ToString();
     }
+
+    /// <summary>
+    // utility function to get the correct text for the ATP label, taking
+    // a number as an input. If greater than 100, it should use F0, otherwise
+    // it should use F1. Above 1000 it should use F0 and add a K.
+    /// </summary>
+    /// <param name="value">The value to format</param>
+    /// <returns>The formatted string</returns>
+    public static string ThreeDigitFormat(double value)
+    {
+        if (value >= 1000)
+            return (value / 1000).ToString("F1", CultureInfo.CurrentCulture) + "K";
```

Comment:
We have a translaton for "K", see https://github.com/Revolutionary-Games/Thrive/blob/eb7b7b1b8039ceed9f7d7d9787c40320316e2fd2/src/general/StringUtils.cs#L37

Reply:
Also, I'm pretty sure we had existing logic for doing this somewhere...

Just looked and there is `StringUtils.FormatNumber`.

Reply:
Thanks, I didn't notice it